src/components/JobCard/__test__/JobCard.test.tsx

```diff
@@ -0,0 +1,101 @@
+import { describe, expect, it, vi } from 'vitest'
+import { render } from '@/tests/customRender'
+import { screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import JobCard from '../JobCard'
+
+// Mock data for the job
+const mockJob = {
+    title: 'Software Engineer',
+    company: 'Tech Corp',
+    location: 'Tokyo, Shibuya',
+    salary: '600,000 - 12,000,000',
+    tags: ['React', 'JavaScript', 'Frontend'],
+    logoUrl: 'https://via.placeholder.com/150',
+    publicationDate: '31/01/2025',
+    jobPostingUrl: 'https://jobpostingurl.com',
+}
+
+describe('JobCard Component', () => {
+    it('should render the job details correctly', () => {
+        render(<JobCard job={mockJob} />)
+
+        // Check the job title
```

Comment:
Let's prefer [self-documenting code](https://en.wikipedia.org/wiki/Self-documenting_code) and only add comments if something is unclear.

```suggestion
```

Please remove similar comments below as they don't add new meaning.